### DISCUSSION TITLE
Linked bases

### DISCUSSION THREAD

**Discussion by Roosteridk** [deta/cloud-docs#586] [Ideas]
It would be nice to be able to reference one base from another. For example, let's say we have Songs and Artists bases. For each song, the artist field could point to an artist. This would solve having to copy the data from the artist to the song. When we want to get the artist field of a particular song, Deta could simply return the associated artist from Artists.

Now you might ask, why not use the artist's key as the field?
That is a possible workaround, however it would be more logical if the database automatically referred us to the original artist without us having to manage keys and also serve as an abstraction to simplify working with Deta as the database grows more complex.

**Comment by jnsougata** [comment]
or you can do one thing ... storing song and artist in the same base and in each artist field make an array for songs and store the ids of the songs the artist is related too. then you can make an OR-ed query to base to get song and artist both in the same time in a single query without using extra base/table.
